DepotManager.cs

```diff
@@ -135,13 +133,8 @@ public class DepotManager {
         long copied = 0;
 
         // @Performance: Start copying while downloads are in progress?
-        foreach (var depot in neededDepots) {
-          var package = Package.Main;
-          if (ManifestData.depots.Contains(depot)) package = Package.Main;
-          else if (depot == ManifestData.GEHENNA) package = Package.Gehenna;
-          else if (depot == ManifestData.PROTOTYPE) package = Package.Prototype;
-
-          CopyAndOverwrite($"{depotLocation}/depot_{depot}", GetFolder(version, package), delegate (long fileSize) {
+        foreach (var manifest in neededManifests) {
+          CopyAndOverwrite($"{depotLocation}/depot_{manifest.depotId}", GetFolder(version, manifest.package), delegate (long fileSize) {

```

Comment:
`depotLocation` is currently hard-coded to the talos appId. Can you update it to use the value from the manifest now?